Messenger/Messenger.Core/Services/TeamService.cs

```diff
@@ -433,8 +433,8 @@ public static async Task<IList<Channel>> GetAllChannelsByTeamId(uint teamId)
         /// </summary>
         /// <param name="role">The name of the role to add</param>
         /// <param name="teamId">The id of the team to add the role to</param>
-        /// <returns>True if successful, false otherwise</returns>
-        public static async Task<bool> AddRole(string role, uint teamId)
+        /// <returns>Id of the added team role</returns>
+        public static async Task<uint> AddRole(string role, uint teamId)
```

Comment:
Make return type nullable to be consistent with the rest of the codebase.

Reply:
resolved in `e58447f`